combineRooms/combineRooms.py

```diff
@@ -0,0 +1,422 @@
+            }
+            await combines_category.set_permissions(guild.default_role, view_channel=is_public, connect=is_public, send_messages=False)
+            await combines_category.set_permissions(league_role, view_channel=True, connect=True, send_messages=False)
+    
+    async def _add_combines_category(self, ctx, name: str):
+        category = await self._get_category_by_name(ctx.guild, name)
+        
+        if category:
+            await ctx.send("A category with the name \"{0}\" already exists".format(name))
+            return None
+        
+        if not await self._is_public_combine(ctx.guild):
+            league_role = self._get_role_by_name(ctx.guild, "League")
+            overwrites = {
+                ctx.guild.default_role: discord.PermissionOverwrite(view_channel=False, connect=False, send_messages=False),
+                league_role: discord.PermissionOverwrite(view_channel=True, connect=True, send_messages=False)
+            }
+        else:
+            overwrites=None
+
+        category = await ctx.guild.create_category(name, overwrites=overwrites)
+        return category
+
+    async def _maybe_remove_combines_voice(self, guild: discord.Guild, voice_channel: discord.VoiceChannel):
+        tier = self._get_voice_tier(voice_channel)
+        category = await self._combines_category(guild)
+        tier_voice_channels = []
+        for vc in category.voice_channels:
+            if tier in vc.name:
+                tier_voice_channels.append(vc)
+        
+        # Never remove the last room for a tier
+        if len(tier_voice_channels) == 1:
+            return False
+
+        # Always retain room 1 for tier:
+        # await voice_channel.delete()
+        i = voice_channel.name.index("room ") + 5
+        # DISABLED: active players in room name
+        # j = voice_channel.name.index(" (")
+        room_num = int(voice_channel.name[i:]) # j])
+        room_one_empty = (room_num == 1)
+
+        # if voice_channel was not room 1
+        if not room_one_empty:
+            # No need to kick scouts. Let them hang out :)
+            if voice_channel.members:
+                return False
+            await voice_channel.delete()
+            return True
+        
+        # delete the other empty room (instead of room 1)
+        for vc in tier_voice_channels:
+            if not vc.members and vc != voice_channel:
+                await vc.delete()
+                return True
+
+    async def _add_combines_info_channel(self, guild: discord.Guild, name: str):
+        category = await self._combines_category(guild)
+        overwrites = {
+            guild.default_role: discord.PermissionOverwrite(send_messages=False)
+        }
+        tc = await category.create_text_channel(name, position=0, permissions_synced=True, overwrites=overwrites)
+
+        acronym = await self._get_acronym(guild)
+        info_message = (
```

Comment:
I think this would be better suited as a customized message we save per guild in the json file rather than a single message to be used across all servers. Maybe make this the default message if the server doesn't have a custom one set. On top of this, since it could be customized I'd remove the acronym and for the default message just use the guild name.

Reply:
I was thinking of something like that, but hadn't explored how to do that yet. In my mind, this would be sent on creation of the combines category, and if an admin wanted, they could replace the message.

Reply:
This can be worked on later for sure. I'd look at how the DE message is saved and used in bulkRoleManager as an example.